### PR TITLE
Update "bring your own cert" docs to add `-days` option for cert expiry

### PR DESCRIPTION
**Pull request by jflowgh**
**Description of the change**
Update docs for "bring your own certs" as per https://github.com/bitnami-labs/sealed-secrets/issues/1126

**Benefits**
Docs work for generating your own certs and sealing them.

**Possible drawbacks**
Need to pick a sane default expiry time for docs

**Applicable issues**
N/A

**Additional information**
N/A

### CODE REVIEW

**Comment by agarcia-oss** [comment]
Thanks for the contribution, @jflowgh ! The changes LGTM; We just need you to include the Sign-off comment in the commit as per the project's [Contributing](https://github.com/bitnami-labs/sealed-secrets/blob/main/CONTRIBUTING.md#technical-requirements) guidelines.

**Comment by josvazg** [comment]
A quick way to sign off one commit:
```bash
git commit --amend -s --no-edit
git push -f
```

**Comment by josvazg** [comment]
Change LGTM too, will approve once the DCO sign-off get sorted out.

**Comment by jflowgh** [comment]
OK the signed off message is there now. Will remember for next time.

**Review by agarcia-oss — APPROVED** [review]
LGTM, thanks!

**Pull request merged**
